README.md

```diff
@@ -57,6 +58,20 @@ export WP_URL=https://edscms.home.blog/
 
 If your blog is private, you can set up `WP_TOKEN` environment variable to pass your access token. To get access token for private WP blog, check out this - https://developer.wordpress.com/docs/oauth2/.
 
+#### CKAN Pages
+
+For more info about enabling and using CKAN Pages:
+https://github.com/ckan/ckanext-pages
+
+By default, the frontend plugin will use the CKAN Pages extension of your enabled CKAN API (as defined by `API_URL` in `.env`).
```

Comment:
Hmm, not sure about that. Can we don't make it default? I think it should be defined explicitly as for WP.

So if you want to use CKAN Pages CMS, you have to define it in the PLUGINS env var.

Reply:
Yes this is the current behavior. It needs to be enabled in `PLUGINS` but by default it will then use the `API_URL`, unless `CKAN_PAGES_URL` is defined (see config.js)